hackathon_site/dashboard/frontend/src/components/teamDetail/TeamCheckoutOrderTable/TeamCheckoutOrderTable.tsx

```diff
@@ -0,0 +1,323 @@
+import {
+    Button,
+    Checkbox,
+    Grid,
+    IconButton,
+    Link,
+    Paper,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+} from "@material-ui/core";
+import { mockHardware, mockCheckedOutOrdersInTable } from "testing/mockData";
+import React, { useState } from "react";
+import Container from "@material-ui/core/Container";
+import styles from "components/general/OrderTables/OrderTables.module.scss";
+import hardwareImagePlaceholder from "assets/images/placeholders/no-hardware-image.svg";
+import MenuItem from "@material-ui/core/MenuItem";
+import Select from "@material-ui/core/Select";
+import {
+    GeneralOrderTableTitle,
+    GeneralOrderTitle,
+} from "components/general/OrderTables/OrderTables";
+import { Formik } from "formik";
+import Info from "@material-ui/icons/Info";
+
+const createDropdownList = (number: number) => {
+    let entry = [];
+
+    for (let i = 0; i <= number; i++) {
+        entry.push(
+            <MenuItem key={i} role="quantity" value={i.toString()}>
+                {i}
+            </MenuItem>
+        );
+    }
+
+    return entry;
+};
+
+const setInitialValues = (
+    request: { id: number; quantityRequested: number; quantityGranted: number }[]
+) => {
+    let orderInitialValues: Record<string, string | boolean> = {};
+    request.forEach((orderItem) => {
+        orderInitialValues[`${orderItem.id}-quantity`] =
+            orderItem.quantityGranted.toString();
+        orderInitialValues[`${orderItem.id}-checkbox`] = false;
+    });
+    return orderInitialValues;
+};
+
+export const TeamCheckoutOrderTable = () => {
+    const orders = mockCheckedOutOrdersInTable;
+    const hardware = mockHardware;
+    const [visibility, setVisibility] = useState(true);
+    const toggleVisibility = () => {
+        setVisibility(!visibility);
+    };
+
+    return (
+        <Container
+            className={styles.tableContainer}
+            maxWidth={false}
```

Comment:
<img width="1877" alt="CleanShot 2022-08-13 at 21 17 33@2x" src="https://user-images.githubusercontent.com/36619969/184518543-218d0416-1407-4749-8b0e-785c0b23aec9.png">

I would suggest to remove this maxWidth prop. The screenshot above is the one without the maxWidth option. I think it looks better tbh. What do you think? And it is under the full-screen mode. 


@YuYing-Liang If you can also check it on your screen and see which one looks better?

Reply:
I checked out dana's branch and fixed the conflicts along the way, but I don't see this problem at all. I checked Dana's css and there's no max-width being set anywhere.

Reply:
it is the in-line css

Reply:
@Leo6Leo could you check again? I still don't see the problem, it might have been fixed already?
![image](https://user-images.githubusercontent.com/20618460/185808357-0448a490-d972-4425-9ef7-199a97f7e45a.png)

Reply:
opps my bad I didn't state that clearly. The picture you sent is the current version we have. I am proposing to change to the one shorter one bcz it looks nicer loll

Reply:
Ohhh okay I'm indifferent to that but if you want to do that, it should be implemented for all the tables, and then we'd also have to make sure the team info table and the team actions table adjust to the smaller width too. Since there are a couple layout changes I think you should make a new UI ticket for it and you can put it at the top of the backlog.

Reply:
sounds good. Then we just leave it unchanged here.